### DISCUSSION TITLE
Not able to render the color data of the OBJ even with MTL files present

### DISCUSSION THREAD

**Discussion by kowshik1234** [pokepetter/ursina#607] [Q&A] [answered]
Hi,

I have created a GUI with the ursina and I am having a very hard time loading the color data of the 3D mode. 

Here's my 3D models and it should render a blue colored milk bottle but it never renders that way.

[3D_model.zip](https://github.com/pokepetter/ursina/files/13207141/3D_model.zip)

What am I missing?

**Comment by pokepetter** [comment]
Triangulate it.

**Reply by kowshik1234** [reply]
I'm sorry, I am new to this. What is triangulating the model please?

**Reply by kowshik1234** [reply, marked as answer]
I worked around this and used the texture solution to create the outer colors. We can close this thread.
